packages/web-components/src/tabs/tabs.base.ts

```diff
@@ -0,0 +1,329 @@
+import { attr, FASTElement, observable } from '@microsoft/fast-element';
+import {
+  keyArrowDown,
+  keyArrowLeft,
+  keyArrowRight,
+  keyArrowUp,
+  keyEnd,
+  keyHome,
+  limit,
+  uniqueId,
+} from '@microsoft/fast-web-utilities';
+import { TabsOrientation } from './tabs.options.js';
```

Comment:
If we want to keep this in base class should any logic that leverges be moved there?

Reply:
Sorry, I'm not tracking - I'm not making API changes here, and everythhing within tabs is used/referenced: 
<img width="591" alt="image" src="https://github.com/user-attachments/assets/e809e60c-9392-458d-95bb-e7cfc6882dfb" />